src/compiler/transformers/module/module.ts

```diff
@@ -1872,8 +1872,8 @@ namespace ts {
                         // Mark the node to prevent triggering this rule again.
                         noSubstitution[getNodeId(expression)] = true;
                         expression = createExportExpression(exportName, expression);
+                        expression = isBinaryExpression(node.parent) ? factory.createParenthesizedExpression(expression) : expression;

```

Comment:
You cannot rely on parents in transformers, prior transformations may not set them so they are likely to be `undefined`.

Reply:
> It would be better to always parenthesize.

@rbuckton Thanks for the feedback. Do you mean that `factory.create Parenthesized Expression (expression)` should be used without a preceding `if` condition? Or do you mean to consider adding changes to the `parenthesizerRules`?